apps/condo/domains/property/components/types.ts

```diff
@@ -0,0 +1,5 @@
+export type TSelectedAddressSuggestion = {
```

Comment:
May be move this part to more specific place? Seems like domain-specific type garbage, which can grows to unreadable scale :)

Reply:
Moved to `apps/condo/domains/property/components/BasePropertyForm`